src/context/AppContext.js

```diff
@@ -92,19 +92,31 @@ function getRoutes(user) {
           linkTo: '/earnings',
           component: EarningsView,
           icon: CreditCardIcon,
-          disabled: process.env.REACT_APP_ENABLE_EARNINGS !== 'true',
```

Comment:
Are the tools and APIs ready for production yet? If not, we should keep the feature flag in so it can be deployed to staging without appearing in production.

Reply:
Okay, Noted. I will return the feature flags until the tool is deployed to production